python/python/glide/async_commands/core.py

```diff
@@ -501,6 +501,9 @@ async def rename(self, key: str, new_key: str) -> TOK:
         meaning that in practice only keys that have the same hash tag can be reliably renamed in cluster.
         See https://redis.io/commands/rename/ for more details.
 
+        Note:
+            When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slot.
```

Comment:
We haven't been using "Note:" in Python since Python doesn't have a @remarks or @apiNote tag. The existing docs just put the warning in its own paragraph, I suggest we do the same. Also need to change `keys` to `key` and `new_key`
```suggestion
        When in cluster mode, the command may route to multiple nodes when `key` and `new_key` map to different hash slots.
```